rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1017,7 +1017,41 @@ private void logUpdateCollections(Transaction rskTx) {
         eventLogger.logUpdateCollections(sender);
     }
 
-    protected void processSVPFailure(Federation proposedFederation) {
+    protected void updateSvpState(Transaction rskTx) {
```

Comment:
If this method is only needed for testing, you can leave it package-private and avoid using `protected`

Reply:
no no, it will be called from `updateCollections`.

Reply:
Yes, but in the meantime it can be package-private. No big deal, resolving this.